templates/bundles/SyliusShopBundle/Homepage/index.html.twig

```diff
@@ -0,0 +1,72 @@
+{% extends '@SyliusShop/layout.html.twig' %}
```

Comment:
Please, update to the newest master and use template events instead of overriding of this template ;)